common/util.go

```diff
@@ -80,6 +80,10 @@ const (
 	retryTaskProcessingMaxInterval     = 100 * time.Millisecond
 	retryTaskProcessingMaxAttempts     = 3
 
+	rescheduleTaskInitialInterval    = 3 * time.Second
+	rescheduleTaskBackoffCoefficient = 1.05
```

Comment:
Shorter initial interval won't help as tasks in the rescheduler will only be rescheduled periodically (batched), instead of maintaining a timer per task and reschedule once the backoff duration is reached. That's why for certain error types, there's this optimization for immediate reschedule when the # of attempts is low.

Back to the coefficient, I don't have a very strong opinion on the exact number. 
For 20 attempts, using 1.05 will give us ~8s backoff, ~20s when using 1.1, ~49s when using 1.15 and ~115s when using 1.2. 
For 30 attempts, ~13s for 1.05, ~52s for 1.1, ~198s for 1.15

I think the backoff calculated with coefficient > 1.1 is kinda too aggressive. 1.05 or 1.1 looks more reasonable to me. Thoughts? We can tune this number as time goes.